op-chain-ops/genesis/config.go

```diff
@@ -69,6 +69,9 @@ type DeployConfig struct {
 	L2GenesisBlockParentHash    common.Hash    `json:"l2GenesisBlockParentHash"`
 	L2GenesisBlockBaseFeePerGas *hexutil.Big   `json:"l2GenesisBlockBaseFeePerGas"`
 
+	// Seconds after genesis block that Regolith hard fork activates. 0 to activate at genesis. Nil to disable regolith
+	L2GenesisRegolithTimeOffset *uint64 `json:"l2GenesisRegolithTimeOffset"`
```

Comment:
This should be a `hexutil.Uint64` so that its standard with the rest of the json serialization/deserialization